NDTensors/test/NDTensorsTestUtils/device_list.jl

```diff
@@ -1,9 +1,10 @@
-using NDTensors: NDTensors
 using Pkg: Pkg
+using NDTensors: NDTensors
+
 if "cuda" in ARGS || "all" in ARGS
   ## Right now adding CUDA during Pkg.test results in a
   ## compat issues. I am adding it back to test/Project.toml
-  # Pkg.add("CUDA")
+  Pkg.add("CUDA")
   using CUDA
```

Comment:
```suggestion
  using CUDA: CUDA
```
to only bring `CUDA` into the namespace.

Reply:
Or maybe it should be:
```julia
  using CUDA: cu
```

Reply:
I am working to fix my workstation currently I cannot test CUDA to verify this code

Reply:
Updated per the comment below. Thanks!